lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -228,42 +228,40 @@ def _get_saved_grade(self, subsection, course_structure, course):  # pylint: dis
                 subsection_grade = SubsectionGrade(subsection)
                 subsection_grade.load_from_data(model, course_structure, self._scores_client, self._submissions_scores)
                 log.warning(
-                    "Loaded saved grade for course with id {0}, version {1}, subsection {2}. grade: {3}".format(
+                    "Persistent Grades: Loaded saved grade for course with id {0}, version {1}, subtree edited on {2}, "
```

Comment:
These should be unicode literals (`u""`).  Otherwise we might get a `UnicodeEncodeError` if non-ascii characters are interpolated.